cockatrice/src/client/ui/picture_loader.cpp

```diff
@@ -136,6 +132,8 @@ PictureLoaderWorker::PictureLoaderWorker()
 #endif
     auto cache = new QNetworkDiskCache(this);
     cache->setCacheDirectory(SettingsCache::instance().getNetworkCachePath());
+    cache->setMaximumCacheSize(1024L * 1024L *
```

Comment:
this is rather confusing to me, not to mention that the L for long int is a different size on different oses and will get cast up to 64bit anyway in the end